parser.go

```diff
@@ -146,6 +146,9 @@ func moveToRoom(exit string) {
 				if curRoom.Name == "Downstairs Hallway" && val.Name == "Staircase" {
 					climbTheStairs()
 				}
+				if curRoom.Name == "Basement Lab" {
+					fmt.Println("There's a message above the door here:\nThe unshrink function only works with the mirror in the attic.\n")
```

Comment:
We're going to hit the user with this every time? Shouldn't it be slightly hidden as a room feature?